.github/workflows/gerrit-info-yaml-verify.yaml

```diff
@@ -0,0 +1,167 @@
+---
+name: Gerrit Info Yaml Verify
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_call:
+    inputs:
+      GERRIT_BRANCH:
```

Comment:
PS add default params for these, so when you try to run the action its all autofilled.

Reply:
There are _no_ defaults that can be set for these. They are always required to be filled in. The reason being that these are handed in from Gerrit and there is no valid default.